src/python/pants/backend/java/lint/google_java_format/BUILD

```
@@ -0,0 +1,8 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+python_sources(dependencies=[":resources"])
+
+python_tests(name="tests")
+
+resources(name="resources", sources=["*.lockfile.txt"])
```

It looks like the only lockfile here is `google_java_format.default.lockfile.txt`? I also agree with Stu that Pants's own internal lockfile should be in `3rdparty/jvm`, not in `src/python/pants/backend` (assuming we can't just the default).

Given that, this is simpler / more atomic:

```suggestion
resource(name="lockfile", source="google_java_format.default.lockfile.txt")
```